frontend/src/routes/trade/ChooseAmount.svelte

```diff
@@ -52,7 +52,7 @@
     <div class="column" />
     <div class="column is-one-third ml-5">
         <header class="title is-3 mb-0">
-            {stock.exchange} : {stock.ticker}
+            {stock.exchange}: {stock.ticker}
```

Comment:
On second thought, I think the ticker is important part, so could maybe do something like `AMZN (XNAS)`? Basically just to make the ticker more prominant.

Reply:
Presentation-wise I think the current one looks neater, so I'm going to stick with what we have? Either way the problem won't be huge at the moment when there's only this many stocks 😂